ui/package.json

```diff
@@ -3,13 +3,16 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@patternfly/react-core": "^2.1.4",
+    "@patternfly/react-table": "^1.0.7",
     "react": "16.8.0",
     "react-dom": "16.8.0",
     "react-redux": "6.0.0",
     "react-scripts": "2.1.3",
     "redux": "4.0.1",
     "redux-logger": "3.0.6",
-    "redux-thunk": "2.3.0"
+    "redux-thunk": "2.3.0",
+    "react-router-dom": "^4.3.1"
```

Comment:
I am afraid that is not a good idea the usage of `ˆ` it will install any upper version which can bring a break change as well. Wdyt over the usage of `~` instead of `ˆ` or a fixed version?

Reply:
@camilamacedo86 Yes happy to make it a fixed version. I installed this using `npm i` but didn't check the version in package.json. Good catch.